models/comments-models.js

```diff
@@ -1,13 +1,49 @@
-const db = require("../db/connection")
+const db = require("../db/connection");
 
 function getCommentsData(article_id, sort_by = "created_at", order = "desc") {
-    return db.query(`SELECT * FROM comments WHERE article_id = $1 ORDER BY ${sort_by} ${order};`, [article_id]).then((comments) => {
-        if (comments.rows.length === 0) {
-            return Promise.reject({status: 404, message: "Article not found"})
-        }
-        return comments.rows
-    })
+  return db
+    .query(
+      `SELECT * FROM comments WHERE article_id = $1 ORDER BY ${sort_by} ${order};`,
+      [article_id]
+    )
+    .then((comments) => {
+      if (comments.rows.length === 0) {
+        return Promise.reject({ status: 404, message: "Article not found" });
+      }
+      return comments.rows;
+    });
 }
 
-module.exports = {getCommentsData}
+function addUsernameData(username, name = "Gurtajs") {
+  return db.query(
+    `INSERT INTO users (username, name) VALUES ($1, $2) RETURNING *`,
+    [username, name]
+  );
```

Comment:
This will add a duplicate user if the username already exists every time you want to post a comment. 👀 Do you want users that aren't valid users to be able to post comments? 🤔